src/app/DownloadsPage.qml

```diff
@@ -16,14 +16,15 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import QtQuick 2.4
-import QtWebEngine 1.5
+import QtQuick 2.9
+import QtWebEngine 1.10
 import Ubuntu.Components 1.3
+import Ubuntu.Components.Popups 1.3
 import Ubuntu.Content 1.3
 import webbrowsercommon.private 0.1
 
 import "MimeTypeMapper.js" as MimeTypeMapper
-import "UrlUtils.js" as UrlUtils
```

Comment:
This PR breaks the ability to open PDF files in the browser directly. The error when tapping the button is:

`file:///usr/share/morph-browser/DownloadsPage.qml:132: ReferenceError: UrlUtils is not defined`

Reply:
Oh you are right, I forgot to remove those codes. I guess because I'm suppose to move them to the content picker dialog in #414

Reply:
I've restored this. I think #414 removes it anyway.